sources/writer.cpp

```diff
@@ -7,6 +7,10 @@
 #include <QTextStream>
 #include <QCoreApplication>
 #include <QDebug>
+#include <QtGlobal>
+#if QT_VERSION >= 0x050A00
```

Comment:
I think we should use here QT_VERSION_CHECK macros instead of magic number 0x050A00.

```
#if (QT_VERSION >= QT_VERSION_CHECK(5, 10, 0))
    #include <QRandomGenerator>
#endif
```